src/Umbraco.Core/Factories/MachineInfoFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Umbraco.Cms.Core.Factories;
+
+public class MachineInfoFactory : IMachineInfoFactory
+{
+    /// <summary>
+    /// Fetches the name of the host machine from the system environment.
+    /// </summary>
+    /// <returns>The name of the host machine.</returns>
```

Comment:
```suggestion
/// <inheritdoc />
```

Might as well inherit docs from the interface :wink: